README.md

```diff
@@ -120,4 +120,4 @@ I would like to thank everyone!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 ## 📝 License
 
-This project is [MIT](./LICENSE.md) licensed.
\ No newline at end of file
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Your MIT license link is not working. Please kindly fix that.
- [ ] Also make sure that `License`, `Acknowledgement`, and `Show your support` topics have the same appearance as the other topics.

Reply:
Fixed with thanks

Reply:
well resolved! ✔️